File: content/polympiads.2/1.introduction-aux-arbres/.codes/empty.cpp
#include <iostream>
#include <vector>

using namespace std;

int main () {
    int nbNoeuds;
    cin >> nbNoeuds;

    vector<int> parents(nbNoeuds);
    for (int i = 0; i < nbNoeuds; i ++) {
        cin >> parents[i];
    }
    
    // Affichez le nombre de feuilles
}

File: content/polympiads.2/1.introduction-aux-arbres/.codes/empty.java
import java.util.Scanner;

class Main { 
	public static void main (String[] args) {
		Scanner sc = new Scanner(System.in);
		int nbNoeuds = sc.nextInt();

		int[] parents = new int[nbNoeuds];
		for (int i = 0; i < nbNoeuds; i ++) {
			parents[i] = sc.nextInt();
		}
		
		// Affichez le nombre de feuilles
	}
}